.github/workflows/test.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       matrix:
         python-version: ["3.9", "3.10"]
         os: ["ubuntu-latest", "macos-latest", "windows-latest"]
-        rdkit-version: ["2022.03", "2022.09"]
+        rdkit-version: ["2022.09", "2023.03.1"]
```

Comment:
Can you set it to `2023.03` everywhere instead? That way the latest `2023.03.*` will always be tested.

Also in the `include` block below, please replace `2022.09` by `2023.03`.